### DISCUSSION TITLE
6.1 issues with Geoserver

### DISCUSSION THREAD

**Discussion by jenw-ua** [ua-gist604b-f24/syllabus#29] [General]
Well, Geoserver is still not uploading or previewing anything for me in 6.0, so I'm attempting 6.1. 

Next issue is with creating the workspace. After configuring the workspace and hitting submit, I again just get a blank white screen. It will not create a workspace.

I've spent 8-10 hours a day every day this week on Module 6 and have had nothing but errors and Geoserver issues. Not really sure what else to do at this point. I've researched, emailed, redone things many times. Day 4 and nothing has improved. About tapped out.

<img width="1500" alt="Screenshot 2024-11-21 at 4 24 24 PM" src="https://github.com/user-attachments/assets/2f464532-c833-430e-9829-299733917332">

White screen of death when creating a workspace.

<img width="1646" alt="Screenshot 2024-11-21 at 4 14 49 PM" src="https://github.com/user-attachments/assets/e4d3937a-7dc5-42a1-94b3-7a5e9c74ec1b">

**Comment by andreasrf** [comment]
Hang in there!! Looks like the Module might be...a little scuffed. I'm hitting day 2 with 6-1 and 6-2 myself. Unfortunately, I am just as stumped as you are about the white screen of death. Consider this me cheering you on for morale. We can do this!!

**Reply by jenw-ua** [reply]
Thanks, I appreciate it!
